homeassistant/components/samsungtv/bridge.py

```diff
@@ -271,12 +279,22 @@ def __init__(
         """Initialize Bridge."""
         super().__init__(method, host, port)
         self.token = token
+        self._app_list: dict[str, str] | None = None
 
     def mac_from_device(self) -> str | None:
         """Try to fetch the mac address of the TV."""
         info = self.device_info()
         return mac_from_device_info(info) if info else None
 
+    def get_app_list(self) -> dict[str, str] | None:
+        """Get installed app list."""
+        if self._app_list is None:
+            if remote := self._get_remote():
+                raw_app_list: list[dict[str, str]] = remote.app_list()
+                self._app_list = {app["name"]: app["appId"] for app in raw_app_list}
```

Comment:
Let's sort it (black will complain about this suggestion…)
```suggestion
                self._app_list = {app["name"]: app["appId"] for app in sorted(raw_app_list, key=lambda app: app["name"])}
```

Reply:
This should be sorted now.